library/RAVE/src/RAVE/audio/IO/SignalConversion.py

```diff
@@ -0,0 +1,9 @@
+from pyodas.core import Stft, IStft
+
+
+class SignalConverter:

```

Comment:
If it's just stft and istft might not be necessary, TBD.

Reply:
Will remove